### PR TITLE
add including scripts in doc

### PR DESCRIPTION
**Pull request by fetrarij**
This part is important, missing inluding theses libs will result an error like #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
...
+      ],
+      "scripts": [
+        "../node_modules/jquery/dist/jquery.min.js",
+        "../node_modules/bootstrap/dist/js/bootstrap.min.js",
+        "../node_modules/moment/min/moment.min.js",
+        "../node_modules/eonasdan-bootstrap-datetimepicker/src/js/bootstrap-datetimepicker.js"
+      ],
+    ```
+    #### Webpack:
+    another example by using [angular webpack described here](https://angular.io/guide/webpack) : import the librairies  in the **vendor.ts** file
```
Comment (atais) [inline README.md]: Are you sure about this config? 
Have you tested it yourself?

File: README.md [hunk 2]
```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
...
+ - eonasdan bootsrap datetimepicker (.js & .css),
+ 
+ - moment.js.
+
+   (nb: you don't need to do another npm install, the previous command already did it)
+    #### Angular cli:
+    an example by using angular cli, you can add theses libs in scripts and styles keys in  **.angular-cli.json**
+
+
+    ```json
```
Comment (atais) [inline README.md]: From what I understand you are using `angular-cli` yourself. So This part is tested on you, right?

File: README.md [hunk 3]
```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
 ```javascript
 npm install ng2-eonasdan-datetimepicker --save
 ```
-
-2) Add the A2Edatetimepicker module import in your module.:
+2) Include all required scripts if it doesn't included yet in your project:
```
Comment (atais) [inline README.md]: There are some typos in the text general and it lacks styling a bit.

I may help you here a bit, when you confirm these configurations work in both scenarios.
Or when https://github.com/atais/ng2-eonasdan-datetimepicker/issues/29 is resolved

File: README.md [hunk 4]
```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
...
+ - jquery.js, 
+ - bootstrap (.js & .css),
+ - eonasdan bootsrap datetimepicker (.js & .css),
+ 
+ - moment.js.
+
+   (nb: you don't need to do another npm install, the previous command already did it)
```
Comment (atais) [inline README.md]: This note is obsolete.

File: README.md (hunk 2 above)
Comment (fetrarij) [reply README.md]: that's right. Both are tested.

File: README.md (hunk 1 above)
Comment (fetrarij) [reply README.md]: Yes, I've tested it before the PR, if you want I can push with examples?

Comment (fetrarij) [reply README.md]: yes, what do you suggest to structure? or having a gh-pages also with a live demo page is useful.

File: README.md (hunk 3 above)
Comment (fetrarij) [reply README.md]: You can edit it :smile: but I can confirm  #29 will be resolved, I tested with webpack & angular 5 with these configurations.

File: README.md (hunk 4 above)
Comment (fetrarij) [reply README.md]: Want me to remove it or you'll do when merging?

File: README.md (hunk 1 above)
Comment (atais) [reply README.md]: just add it in `/examples/webpack` and `/examples/angular-cli` subfolders

Comment (atais) [reply README.md]: also ignore it in package.json.

File: README.md (hunk 2 above)
Comment (atais) [reply README.md]: OK I will fix it and merge this changes.

File: README.md (hunk 3 above)
Comment (atais) [reply README.md]: OK I will fix it and merge this changes.

File: README.md (hunk 4 above)
Comment (atais) [reply README.md]: i will do it, thanks for all the support.

File: README.md (hunk 1 above)
Comment (fetrarij) [reply README.md]: what do you prefer for ignoring folder? using .npmignore or by specifiyng all including files in files?

Comment (atais) [reply README.md]: either is fine

**Pull request merged**